### ISSUE TITLE
Branch policy for rebase strategy

### ISSUE THREAD

**Issue by tiasmc** [MicrosoftDocs/vsts-docs#411]
When running the rebase strategy we are required to rebase the pull request to the newest version before completing the pull request. When we sometimes forget to rebase before completing the history for the master branch gets hard to follow.
Can we create a branch policy to require pull requests being at the latest version of the target branch before completion?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bcf2704e-ef04-e4ab-88ba-48533fa62d21
* Version Independent ID: e59a9067-775b-d9df-94b7-29076dbfbf25
* Content: [Update your branch history with rebase](https://docs.microsoft.com/en-us/vsts/git/tutorial/rebase?view=vsts&tabs=visual-studio)
* Content Source: [docs/git/tutorial/rebase.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/git/tutorial/rebase.md)
* Service: **unspecified**
* GitHub Login: @douge
* Microsoft Alias: **sdanie**

**Comment by steved0x** [comment]
@tiasmc Branch policies don't support this option currently, but you can make a feature request on the [UserVoice for VSTS](https://developercommunity.visualstudio.com/spaces/21/index.html) to let the team know that is a feature that you need. The team uses these feature requests (and the votes they may receive) to help determine and prioritize future work. Thank you.
